### DISCUSSION TITLE
When do we plan to tag and release 4.2.4 ?

### DISCUSSION THREAD

**Discussion by atulsinghlw** [xetorthio/jedis#3068] [Q&A]
Need to use StreamEntryID.MINIMUM_ID, StreamEntryID.MAXIMUM_ID.
Wondering when are we planning to tag and release 4.2.4 ?

**Comment by sazzad16** [comment]
There is no current plan for 4.2.4. 

Your concerned features will be included in 4.3.0. Until it is released, you may consider using 4.3.0 snapshot. [Here](https://github.com/redis/jedis/blob/master/docs/jedis-maven.md) is a hint on using Jedis snapshot.
